contrib/labstack/echo.v4/option.go

```diff
@@ -126,3 +128,15 @@ func WithHeaderTags(headers []string) Option {
 		cfg.headerTags = internal.NewLockMap(headerTagsMap)
 	}
 }
+
+// WithTag sets a key/value pair that should be set as metadata on the new span.
+// The custom tag will be set when the span starts and can be replaced by
+// default tags.
+func WithTag(key string, value interface{}) Option {
```

Comment:
Should I rename this to `WithCustomTag` to follow the pattern from other places (e.g. `grpc`)?

https://github.com/DataDog/dd-trace-go/blob/9d39644c0e722861d594f6c55970216ca1096de8/contrib/google.golang.org/grpc/option.go#L202-L210